mod_reforged/hooks/skills/racial/flesh_golem_racial.nut

```diff
@@ -0,0 +1,34 @@
+::Reforged.HooksMod.hook("scripts/skills/racial/flesh_golem_racial", function(q) {
+	q.m.InjuryThresholdMult <- 0.7;
+	q.m.MovementAPCostModifier <- 2;
+
+	q.create = @(__original) function()
+	{
+		__original();
+		this.m.Name = "Flesh Golem";
+		this.m.Icon = "ui/orientation/flesh_golem_orientation.png";
+		this.m.IsHidden = false;
+		if (this.isType(::Const.SkillType.Perk))
+			this.removeType(::Const.SkillType.Perk);	// This effect having the type 'Perk' serves no purpose and only causes issues in modding
+	}
+
+	q.getTooltip <- function()
```

Comment:
Use `getTooltip = @() function()`, because the `getTooltip` function exists in a parent of this object.

Reply:
This was somehow resolved but the changes were not implemented.